sciLifeLab_utils/run_validation.py

```diff
@@ -76,7 +76,7 @@ def main(args):
             sample_YAML.close
 
             # Run the job
-            extramodules = ["module load samtools\nmodule load bwa\nmodule load BUSCO/1.1b1\n"]
+            extramodules = ["module load samtools\nmodule load bwa\nmodule load BUSCO/1.22\nsource $BUSCO_SETUP\n"]
```

Comment:
ah ha,. but user should have the environment variable `$BUSCO_SETUP` ready while submitting the jobs I suppose ?